docs/config.rst

```diff
@@ -48,6 +48,11 @@ core
 
     If ``false``, Gordon will exit out if it can't load one or more plugins.
 
+.. option:: route=DICT-OF-STRINGS
```

Comment:
Totally guessing here. I wasn't able to find a page that described how to format these by googling alone.  Anybody have a reference to recommend?